ql/src/java/org/apache/hadoop/hive/ql/exec/repl/AtlasDumpTask.java

```diff
@@ -73,7 +74,12 @@ public int execute() {
       String entityGuid = checkHiveEntityGuid(atlasRequestBuilder, atlasReplInfo.getSrcCluster(),
               atlasReplInfo.getSrcDB());
       long currentModifiedTime = getCurrentTimestamp(atlasReplInfo, entityGuid);
-      dumpAtlasMetaData(atlasRequestBuilder, atlasReplInfo);
+      AtlasDumpLogger replLogger = new AtlasDumpLogger(atlasReplInfo.getSrcDB(),
+              atlasReplInfo.getStagingDir().toString());
+      replLogger.startLog();
```

Comment:
start should be called when the atlas dump task starts. This will give the complete estimate of time taken.